rrfs-test/validated_yamls/templates/obtype_config/atms_n20.yaml

```diff
@@ -0,0 +1,585 @@
+     - obs space:
+         name: atms_n20
+         distribution:
+           name: "@DISTRIBUTION@"
+         obsdatain:
+           engine:
+             type: H5File
+             obsfile: data/obs/ioda_atms_n20.nc
+         obsdataout:
+           engine:
+             type: H5File
+             obsfile: atms_n20_hofx.nc4
+         simulated variables: [brightnessTemperature]
+         observed variables: [brightnessTemperature]
+         channels: &atms_n20_channels 1-22
+       obs operator:
+           name: CRTM
+           Absorbers: [H2O,O3,CO2]
+           Clouds: [Water, Ice]
```

Comment:
HI Xiaoyan, did you see any differences with and without adding cloud inputs?

Reply:
@xyzemc Thanks for the update!  So the assimilated numbers for channels 8-13 (with  channels 14 and 15 turned off?) are the same with and without cloud inputs? Channels 12/13 are higher channels than channels 1-11, do you have any thoughts why the assimilated numbers are much smaller for channel 12/13, especially for channel 13, compared to channels 1-11? Thanks!

Reply:
@HaidaoLin-NOAA  Yes, channel 14 and 15 are turned off.
The data count passed QC for channel 8-13 are same. I plot the simulated brightness temperature difference between with & without cloud input, which are zero. But there are difference for channel 1-7 simulated brightness temperature. I will put the difference plots here.
Good catch for channel 12-13 has less data passed the QC. I have no clue for that.

The difference of simulated brightness temperature (hofx) between with cloudy inputs and clear inputs:
![image](https://github.com/user-attachments/assets/67035c9c-cbf4-40c6-8703-162d7fd00144)
![image](https://github.com/user-attachments/assets/d261434b-7f77-4f18-8376-4fa1d0173cb3)
![image](https://github.com/user-attachments/assets/6ee166dd-f019-47f8-a76f-0c56e554a24d)
![image](https://github.com/user-attachments/assets/19a468e2-cef6-4aa3-a6d2-1955357426a8)
![image](https://github.com/user-attachments/assets/7fc2930c-8750-4d32-9004-d955734a6752)
![image](https://github.com/user-attachments/assets/d11fc4ae-da9d-485c-9c4a-282b618d8bc3)
![image](https://github.com/user-attachments/assets/2e34f1a3-7d64-4a08-8794-4fcb3e54c5f8)

Reply:
@xyzemc Thank you for the update and plots!  I have a few questions here:  
1. There are some coverages/colors in the left, looks like they are not within the model domain, are they observation locations outside the model domain?  What's the color means? 
2. Why for channels 6/7, before QC, the total counts are only 4031 and 88, but for channels 1-5, they are over 9000?
3.  For channel 7, looks like there are only 88 observation counts?  If so, what's the meaning for the color for other areas in the ATMS swath in the domain or just indicating missing values? 
Sorry for more questions, just trying to understand more.  We can also talk more on this offline. Thanks!

Reply:
@HaidaoLin-NOAA Thanks for you checking it carefully.
1. The coverage on the left with light blue core means the data outside of the domain, this value is missing value. So that means after online domain check, although those data marked with QC=12, but it still in the memory filled with missing value.
2 & 3. You are right. All the channels should have same data count before QC. I found I plotted the difference of simulated brightness temperature wrong in these figures. I tried to filter out the data point outside of the domain having a missing value brightness temperature, but I accidently used the difference of brightness temperature. With the bug fix in my plotting python code. Here are correct plots to explain this:

**Channel dependent difference of simulated brightness temperature with cloud input and clear input**
![image](https://github.com/user-attachments/assets/072c47d5-0779-4b16-86fb-8e60eb4e2c3c)

Reply:
@xyzemc Thank you for the update and nice plots!  I don't need to see more plots now, just two more comments for possible future reference: 1) checking only the data passing the thinning, this would reduce a lot of data points and would be easier to see the differences due to other QC; 2) figuring out which QC is actually removing the data for cloud/clear inputs.  Nice job and thanks!

Reply:
Thanks for your comments. That is very helpful to understand how the QC filter work for the cloudy radiance.